src/coreComponents/linearAlgebra/utilities/LAIHelperFunctions.hpp

```diff
@@ -247,7 +247,7 @@ void computeRigidBodyModes( MeshLevel const & mesh,
   array1d< globalIndex > globalNodeList;
   for( localIndex k = 0; k < LvArray::integerConversion< localIndex >( selection.size() ); ++k )
   {
-    if( dofManager.getLocation( selection[k] ) == DofManager::Location::Node )
+    if( dofManager.location( selection[k] ) == DofManager::Location::Node )
```

Comment:
The problem with this 
```suggestion
    if( dofManager.getLocation( selection[k] ) == DofManager::Location::Node )
```
is that with the `get` prefix, without knowing what I am talking about, I know that I am probably accessing some immutable component (or something like that) of `location`. 
Without `get`, it's not obvious anymore if I am setting `location` to a certain value and then use some return type while passing through. It's an additional cost for new comers.

Reply:
In this case I just made it consistent with other member functions that didn't use `get` prefix (`numComponents()`, `rankOffset()`, etc.). I also just realized there is still one inconsistent member `getKey()` left, but... "we'll get'em next time".

In general, I prefer `thing()` and `setThing()` for getter and setter, respectively. I think any mutating member function should contain an action verb, and a lack thereof signifies a "passive" getter or some kind of const inspection method. This seems to me like a pretty intuitive convention (though exceptions are possible, of course).

(For the record, I totally used to be in the `get`/`set` camp. My first OOP language was Java in ~2006, and this convention was prevalent there at the time. I think I shifted over time, especially as I looked more and more at  modern C++ code bases and style guides).

Reply:
OK then!

Personally I am in the _"think less"_ side. `get`, `set` and we're all set 😉 .
Also, when in the IDE, you can just type `get` and ask for completion. That's really convenient when you do not know where you are.